lib/tests/s3-upload.test.js

```diff
@@ -0,0 +1,37 @@
+import fs from "fs";
+import { uploadFileToS3, readStream2buffer } from "../../pages/api/s3-upload";
+
+let mockUuid = jest.fn(() => "14528745455");
+
+jest.mock("uuid", () => {
+  return { uuid: () => mockUuid };
+});
+
+describe("S3Client", () => {
+  it("Should raise an error with malformed parameters", async () => {
+    jest.spyOn(global, "Date").mockImplementationOnce(() => new Date("2021-08-12"));
+    await uploadFileToS3(Buffer.from("test"), "temp-s3-upload-testing", "test").catch((error) => {
+      expect(error).toBeTruthy();
+    });
+  });
+});
+
+describe("uploadFileToS3", () => {
+  it("Should call upload method with incorrect file path", async () => {
+    jest.spyOn(global, "Date").mockImplementationOnce(() => new Date("2021-08-12"));
+
+    await uploadFileToS3({ path: "./test" }, "temp-s3-upload-testing", "test").then((data) => {
```

Comment:
Modified the test to check for the bad file path.  Passing in a Buffer object was throwing an error because `path` did not exist on the Buffer object.